rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -596,3 +597,41 @@ def test_augmented_detection_pendle_transactions(
 
     updated_token = cast(EvmToken, globaldb.resolve_asset(identifier=token.identifier))
     assert updated_token.protocol is None
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('max_tasks_num', [5])
+def test_maybe_update_aave_v3_underlying_assets(
+        task_manager: TaskManager,
+        globaldb: GlobalDBHandler,
+) -> None:
+    """Test that the task updating the aave v3 underlying assets in globaldb works"""
+
+    # check the aave v3 underlying assets table in globaldb before running the task
+    with globaldb.conn.write_ctx() as write_cursor:
+        write_cursor.execute(  # remove all the aave-v3 tokens
+            'DELETE FROM assets WHERE identifier IN '
+            '(SELECT identifier FROM evm_tokens WHERE protocol = ?);',
+            (CPT_AAVE_V3,),
+        )
+
+        assert write_cursor.execute(
+            'SELECT COUNT(*) FROM evm_tokens WHERE protocol = ?;', (CPT_AAVE_V3,),
+        ).fetchone()[0] == 0
+
+    task_manager.potential_tasks = [task_manager._maybe_update_aave_v3_underlying_assets]
+    task_manager.schedule()
+    gevent.joinall(task_manager.running_greenlets[task_manager._maybe_update_aave_v3_underlying_assets])  # wait for the task to finish since it might context switch while running  # noqa: E501
+
+    # check the aave v3 underlying assets table in globaldb after running the task
+    with globaldb.conn.read_ctx() as write_cursor:
+        assert write_cursor.execute(
+            'SELECT COUNT(*) FROM evm_tokens WHERE protocol = ?;',
+            (CPT_AAVE_V3,),
+        ).fetchone()[0] > 0
+
+        assert write_cursor.execute(
+            'SELECT COUNT(*) FROM underlying_tokens_list WHERE parent_token_entry IN '
+            '(SELECT identifier FROM evm_tokens WHERE protocol = ?);',
+            (CPT_AAVE_V3,),
+        ).fetchone()[0] > 0
```

Comment:
Please also add a check for 1-2 specific tokens on all the different EVM chains.